Hush/Assets/Scripts/Relics/RelicDome.cs

```diff
@@ -16,4 +32,31 @@ public class RelicDome : MonoBehaviour
     public void OpenDome() {
         gameObject.SetActive(false);
     }
+
+    public void TakeDamage(int damage)
+    {
+        hitPoints -= damage;
+        if (hitPoints <= 0)
+        {
+            hitPoints = 0;
+            Die();
+        }
+        if (hitPoints < maxHitPoints)
+        {
+            OnDestroyDome();
```

Comment:
Created a bug for this. Staying in for now because reasons

Reply:
#59